### DISCUSSION TITLE
Leveraging D3 to create interactive visualisations in cQube

### DISCUSSION THREAD

**Discussion by abhineetsingh10** [Sunbird-cQube/community#71] [Ideas]
I am sharing a sample that shows the total students by State, District and School Type from the UDISE 2019 data. This visualisation has been created in D3 which is an open-source tool to create amazing interactive visualization. The code is free for anyone to use and customise as per their requirements. 

Link to access the visual- https://vizhub.com/abhineetsingh10/7dea3d7db5924655bd98e418283befaa

Link to access the code- https://vizhub.com/abhineetsingh10/7dea3d7db5924655bd98e418283befaa?edit=files&file=index.html

Link to access the data- https://vizhub.com/abhineetsingh10/7dea3d7db5924655bd98e418283befaa?edit=files&file=udise.json

**Comment by surendrasinghs** [comment]
Thanks, @abhineetsingh10. 

+1 to the idea. Treemap is one of the widely used interactive drill-down visualisations for location/hierarchical data. I would assume/imagine that as the user drills down it would be useful to show many/multiple metrics for the selected location. Just seeing one high-level metric may not give much value to the user. This is a good start to interactive drill-down visualisations - let's bring in the actionability aspects.
